piccolo/apps/user/commands/list.py

```diff
@@ -0,0 +1,26 @@
+from piccolo.apps.user.tables import BaseUser
+
+
+def list():
+    """
+    List existing users.
+    """
+    users = BaseUser.select(
+        BaseUser.all_columns(exclude=[BaseUser.password])
+    ).run_sync()
```

Comment:
We can sort the results - I guess `username` is most useful?

Reply:
Done.